tests/functional/spec/date_reference_ranges.spec.js

```diff
@@ -0,0 +1,41 @@
+const helpers = require('../helpers');
+
+const ManualRangeBlockPage = require('../pages/features/date_reference_ranges/manual-range-block.page.js');
+const DateSeparateBlockPage = require('../pages/features/date_reference_ranges/date-separate-block.page.js');
+const DateRangeBlockPage = require('../pages/features/date_reference_ranges/date-range-block.page.js');
+const SummaryPage = require('../pages/features/date_reference_ranges/summary.page.js');
+
+describe('Reference Date Range Checks', function() {
+  // We have no way to mock out started_at, the unit and integration tests cover the date values
+  // this test is only to validate the general questionnaire functionality and to make sure
+  // the date format is correct.
+  it('Given we start the survey, when we click the button, the dates should be in the correct format...', function() {
+    return helpers.startQuestionnaire('test_date_reference_ranges.json').then(() => {
```

Comment:
I think you need to change this to `openQuestionnaire` as it doesn't have intro.

Reply:
I did run the tests after the change, and it did work. I'll switch it over though.

Reply:
Apparently I didn't :)